automated/linux/armnn/armnn-unit-tests.sh

```diff
@@ -24,20 +24,18 @@ pkgs="ntp wget gcc g++ python3 python3-pip"
 dhclient
 install_deps "${pkgs}" "${SKIP_INSTALL}"
 
-#(GET WHICH PLATFORM)
-detect_abi
-
-if [ "$abi" == "armeabi" ]
-then
-    mkdir /usr/local/bin && cd /usr/local/bin && \
-    wget "${LINK_SNAPSHOT_32BIT}" && tar xf armnn-32bit.tar.xz && \
-    export BASEDIR="/home/buildslave/workspace/armnn32-ci-build"
-else
-    wget "${LINK_SNAPSHOT_64BIT}" && tar xf armnn.tar.xz && \
-    export BASEDIR="home/buildslave/workspace/armnn-ci-build"
-
-cd $BASEDIR/build
-ln -s libprotobuf.so.15.0.0 ./libprotobuf.so.15
-export LD_LIBRARY_PATH=`pwd`
-chmod a+x UnitTests
-./UnitTests
+if [[-z ${ARMNN_TARBALL} ]]; then
```

Comment:
I'm so sorry, this is my fault... I think it should be '-n' and not '-z'.
       -n STRING
              the length of STRING is nonzero
       -z STRING
              the length of STRING is zero

Reply:
Ah yeah you're right, good catch